README.md

```diff
@@ -1 +1,64 @@
-# Bucketeer CLI
+# Bucketeer CLI &nbsp;[![Build Status](https://travis-ci.com/UCLALibrary/bucketeer-cli.svg?branch=main)](https://travis-ci.com/UCLALibrary/bucketeer-cli)
+
+Uploads CSV files to the [Bucketeer](https://github.com/UCLALibrary/bucketeer) service for processing.
+
+## Installation
+
+First, ensure that you have Bash, cURL, Python 3.6+ and Pip installed on your system.
```

Comment:
And Black and PyTest? Do we want to give instructions for these last two?

Reply:
I'd argue against mentioning black and pytest here since they're dev dependencies

Reply:
Okay.